docs/Authentication.md

```diff
@@ -7,13 +7,13 @@ Kusto Spark connector allows the user to authenticate with AAD using an AAD appl
 This authentication method is fairly straightforward, and it is used in most of the examples in this documentation.
 
  * **KUSTO_AAD_CLIENT_ID**: 
-  AAD application (client) identifier.
+  'kustoAADClientID' - AAD application (client) identifier.
   
  * **KUSTO_AAD_AUTHORITY_ID**: 
-  AAD authentication authority. This is the AAD Directory (tenant) ID.
+  'kustoAADAuthorityID' - AAD authentication authority. This is the AAD Directory (tenant) ID.
  
  * **KUSTO_AAD_CLIENT_PASSWORD**: 
- AAD application key for the client.
+  'kustoClientAADClientPassword' - AAD application key for the client.
```

Comment:
Actually, this is either a password (for user) or app secret (for app). The names should actually be aligned to the scenario.